functions/EmuScripts/emuDeckCitra.sh

```diff
@@ -15,7 +15,8 @@ Citra_finalize(){
 Citra_install(){
 	setMSG "Installing $Citra_emuName"	
 	installEmuFP "${Citra_emuName}" "${Citra_emuPath}"	
-	flatpak override "${Citra_emuPath}" --filesystem=host --user	
+	curl -L https://github.com/citra-emu/citra-web/releases/download/2.0/citra-setup-linux > citra-setup-linux && chmod +x citra-setup-linux && ./citra-setup-linux --accept-licenses --confirm-command install
+	rm citra-setup-linux
```

Comment:
The binupdate.sh needs to be updated also, so that emulator can be updated. Usually the _Install method is called from binupdate so this code should be modified to handle updates.